torchprime/experimental/torchax_models/run.py

```diff
@@ -134,15 +134,20 @@ def register_attention(fn):
   env = torchax.default_env()
   k = torch.nn.functional.scaled_dot_product_attention
   env._ops[k] = ops_registry.Operator(
-    k, fn, is_jax_function=False, is_user_defined=True, needs_env=False
+    k,
+    fn,
+    is_jax_function=False,
+    is_user_defined=True,
+    needs_env=False,
+    is_view_op=False,
   )
 
 
 def make_weight_shard(weight_meta, slice_index):
   weight_shard_meta = weight_meta[slice_index]
   with torchax.default_env():
     return interop.jax_view(
-      torch.randn(weight_shard_meta.shape, dtype=weight_shard_meta.dtype)
+      torch.randn(weight_shard_meta.shape, dtype=weight_shard_meta.dtype) * 0.1
```

Comment:
just a question, should it be xavier or kaiming instead of randn?

Reply:
probably, adding *0.1 fixed the overflow issue so i just add that simple fix.